src/pages/ReportDetailsPage.tsx

```diff
@@ -83,18 +83,24 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
 
     const isPrivateNotesFetchTriggered = report?.isLoadingPrivateNotes !== undefined;
 
+    const isSelfDM = useMemo(() => ReportUtils.isSelfDM(report), [report]);
+
     useEffect(() => {
-        // Do not fetch private notes if isLoadingPrivateNotes is already defined, or if network is offline.
-        if (isPrivateNotesFetchTriggered || isOffline) {
+        // Do not fetch private notes if isLoadingPrivateNotes is already defined, or if network is offline or if report is a self DM
+        if (isPrivateNotesFetchTriggered || isOffline || isSelfDM) {
             return;
         }
 
         Report.getReportPrivateNote(report?.reportID ?? '');
-    }, [report?.reportID, isOffline, isPrivateNotesFetchTriggered]);
+    }, [report?.reportID, isOffline, isPrivateNotesFetchTriggered, isSelfDM]);
 
     const menuItems: ReportDetailsPageMenuItem[] = useMemo(() => {
         const items: ReportDetailsPageMenuItem[] = [];
 
+        if (isSelfDM) {
+            return items;
+        }
```

Comment:
I would recommend using plain [] just to be error-proof for future code changes.

```suggestion
        if (isSelfDM) {
            return [];
        }
```